evalml/automl/automl_search.py

```diff
@@ -267,9 +268,9 @@ def __init__(self,
         # make everything ww objects
         self.X_train = _convert_to_woodwork_structure(X_train)
         self.y_train = _convert_to_woodwork_structure(y_train)
-
+        self.sampler = sampler if (sampler is not None and sampler != 'None') else None
```

Comment:
'None' is primarily for LG test usage. I updated a LG branch to take in the ['sampler' arg](https://github.com/alteryx/evalml_looking_glass/compare/bc_imbalance_sampling#diff-96b21daac4fea193abcbda0eed3e0a0f89bf31eef2dc2593dd8b1749369a8acfR24)

Reply:
Could we move the 'None' value into LG instead? Let's keep this logic as simple as we can.